packages/labs/src/components/select/select.tsx

```diff
@@ -248,8 +248,9 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
         Utils.safeInvoke(popoverProps.popoverWillClose);
     }
 
-    private handleQueryChange = (event: React.FormEvent<HTMLInputElement>) => {
-        this.setState({ query: event.currentTarget.value });
+    private handleQueryChange = (e: React.FormEvent<HTMLInputElement>) => {
```

Comment:
`e` makes sense for the no-conflict `window.event` argument. all i care about is consitency